plugins/woocommerce/client/admin/client/settings-payments/settings-payments-main.tsx

```diff
@@ -207,16 +210,36 @@ export const SettingsPaymentsMain = () => {
 					incentive,
 					'wc_settings_payments__banner'
 				)
+			) {
+				const referenceTimestamp = new Date();
+				referenceTimestamp.setDate( referenceTimestamp.getDate() - 30 );
```

Comment:
It's a requirement from Tim, see the issue description. Not ideal implementation, but it works. Curious about your feedback! Added a comment below that it has similar logic on the server for red dot notice.

Reply:
During the chat with @vladolaru we agreed that this tradeoff is fine for now, implementing it on the backend would be tricky and we don't see much benefits for that ATM